doc/vm-creation-example.md

```diff
@@ -0,0 +1,106 @@
+# Example - Allocating MAC addresses to a VirtualMachine
+
+## Create a NetworkAttachmentDefinition:
+
+The NetworkAttachmentDefinition is used to setup the network attachment, i.e. secondary interface for the VirtualMachine.
+For more information see [Kubernetes Network Custom Resource Definition De-facto Standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit).
+This example will use a linux-bridge.
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+ name: br1
+spec:
+  config: '{
+    "cniVersion": "0.3.1",
+    "name": "br1",
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br1"
+        },
+        {
+            "type": "tuning"
+        }
+      ]
+    }'
+```
+
+**Note:** the tuning plugin changes the MAC address after the main plugin was executed. Make sure
+the main plugin configure does not have MAC filter on the interface.
+
+## Allocate a MAC addresses to a VirtualMachine
+
+Create the VirtualMachine with 1 secondary NIC:
+
+```bash
```

Comment:
Use yaml, it will make most of this example pretty

Reply:
Done